src/pages/MigrateV1/index.tsx

```diff
@@ -17,14 +17,27 @@ import { EmptyState } from './EmptyState'
 import V1PositionCard from '../../components/PositionCard/V1'
 import QuestionHelper from '../../components/QuestionHelper'
 import { Dots } from '../../components/swap/styleds'
+import { useAddUserToken } from '../../state/user/hooks'
+import useIsDefaultToken from '../../hooks/useIsDefaultToken'
+import useIsCustomAddedToken from '../../hooks/useIsCustomAddedToken'
 
 export default function MigrateV1({ history }: RouteComponentProps) {
   const theme = useContext(ThemeContext)
   const { account, chainId } = useActiveWeb3React()
 
   const [tokenSearch, setTokenSearch] = useState<string>('')
   const handleTokenSearchChange = useCallback(e => setTokenSearch(e.target.value), [setTokenSearch])
-  useToken(tokenSearch)
+
+  // automatically add the search token
```

Comment:
why do we automatically add here?

Reply:
otherwise if they refresh or page forward/backward or try to add multiple custom tokens the entry disappears, which feels a bit weirder than automatically adding them imo.